messaging/artemis/src/test/resources/amq.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: amq-broker
+  type: Opaque
+  stringData:
+    username: quarkus
+    password: quarkus
+    queues: prices
+
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: amq-broker
+  spec:
+    lookupPolicy:
+      local: false
+    tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: registry.access.redhat.com/amq-broker-7/amq-broker-72-openshift
```

Comment:
What if I want AMQ 7.6? I can just rewrite it here I guess.

Reply:
7.6 is behind auth, so I went with 7.2, the latest that's accessible without auth. Changing this single line should be enough, yea.

Reply:
Thanks for confirmation.